x-pack/plugins/security_solution/server/endpoint/lib/artifacts/lists.ts

```diff
@@ -124,12 +132,17 @@ function translateItem(
   schemaVersion: string,
   item: ExceptionListItemSchema
 ): TranslatedExceptionListItem {
+  const itemSet = new Set();
```

Comment:
Again you could use a custom accumulator here to avoid the global state... not sure how much of a problem it is, maybe get another opinion on it. Looks like it will work though.

Reply:
Not too well versed on this. Does this provide us with benefits other than syntactic sugar/style?

Reply:
@alexk307 I'm not the best authority on javascript, but my understanding is that it's best practice to avoid mutable state and mixed scope. This might not be so bad because `itemSet` is a const, but it's introducing a block scope variable that's (probably unnecessarily?) external to the scope of the reduce function.

Reply:
And this too.